lib/shopify-cli/commands/connect.rb

```diff
@@ -6,7 +6,7 @@ class Connect < ShopifyCli::Command
       def call(*)
         project_type = ask_project_type unless Project.has_current?
 
-        if Project.has_current? && Project.current
+        if Project.has_current? && Project.current && Project.current.env
```

Comment:
How come this was added? I don't think a project _needs_ to have an env to be considered already connected, right? Some project types might not need an env but still may be connected

Reply:
Spoke with Jacob, and it looks like connect always creates an .env file, so I will leave this change in for now. The motivation behind this is so that the message `! This app appears to be already connected` doesn't appear when the project doesn't have an .env file already.